lib/OrdinaryDiffEqRKC/src/OrdinaryDiffEqRKC.jl

```diff
@@ -0,0 +1,53 @@
+module OrdinaryDiffEqRKC
+
+import OrdinaryDiffEq: alg_order, alg_maximum_order, get_current_adaptive_order,
+                       get_current_alg_order, calculate_residuals!, accept_step_controller,
+                       default_controller, beta2_default, beta1_default, gamma_default,
+                       initialize!, perform_step!, @unpack, unwrap_alg, isthreaded,
+                       step_accept_controller!, calculate_residuals,
+                       OrdinaryDiffEqMutableCache, OrdinaryDiffEqConstantCache,
+                       reset_alg_dependent_opts!, AbstractController,
+                       step_accept_controller!, step_reject_controller!,
+                       OrdinaryDiffEqAdaptiveAlgorithm, OrdinaryDiffEqAdaptiveImplicitAlgorithm,
+                       alg_cache, CompiledFloats, @threaded, stepsize_controller!, DEFAULT_PRECS,
+                       constvalue, PolyesterThreads, Sequential, BaseThreads,
+                       _digest_beta1_beta2, timedepentdtmin, _unwrap_val,
+                       TimeDerivativeWrapper, UDerivativeWrapper, calc_J, _reshape, _vec,
+                       WOperator, TimeGradientWrapper, UJacobianWrapper, build_grad_config,
+                       build_jac_config, calc_J!, jacobian2W!, dolinsolve 
+using DiffEqBase, FastBroadcast, Polyester, MuladdMacro, RecursiveArrayTools, LinearSolve
+import OrdinaryDiffEq: ROCK2, ROCK4, RKC, ESERK4, ESERK5, SERK2, IRKC
+
+macro cache(expr)
```

Comment:
Note for @oscardssmith , I'm not 100% sure why this macro needs to be per-module, but I had to do this in the extrapolation one too so we can merge like this and figure it out later.